Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -246,7 +246,7 @@ def fetch_mails(client: IMAPClient,
             continue
         # The search query filters emails by day, not by exact date
         email_message_object = Email(message_bytes, include_raw_body, save_file, mail_id)
-        if not first_fetch_time or email_message_object.date > first_fetch_time:
+        if email_message_object.date > first_fetch_time:
```

Comment:
`first_fetch_time` can be none why removing the condition?

Reply:
how can it be None?